pkg/ride/functions_proto.go

```diff
@@ -1248,89 +1183,6 @@ func ecRecover(_ environment, args ...rideType) (rideType, error) {
 	return rideBytes(pkb[1:]), nil
 }
 
-func checkedBytesDataEntry(_ environment, args ...rideType) (rideType, error) {
-	if err := checkArgs(args, 2); err != nil {
-		return nil, errors.Wrap(err, "checkedBytesDataEntry")
-	}
-	key, ok := args[0].(rideString)
-	if !ok {
-		return nil, errors.Errorf("checkedBytesDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	value, ok := args[1].(rideBytes)
-	if !ok {
-		return nil, errors.Errorf("checkedBytesDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	return newRideBinaryEntry(
-		key,
-		value,
-	), nil
-}
-
-func checkedBooleanDataEntry(_ environment, args ...rideType) (rideType, error) {
-	if err := checkArgs(args, 2); err != nil {
-		return nil, errors.Wrap(err, "checkedBooleanDataEntry")
-	}
-	key, ok := args[0].(rideString)
-	if !ok {
-		return nil, errors.Errorf("checkedBooleanDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	value, ok := args[1].(rideBoolean)
-	if !ok {
-		return nil, errors.Errorf("checkedBooleanDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	return newRideBooleanEntry(
-		key,
-		value,
-	), nil
-}
-
-func checkedDeleteEntry(_ environment, args ...rideType) (rideType, error) {
-	key, err := stringArg(args)
-	if err != nil {
-		return nil, errors.Wrap(err, "checkedDeleteEntry")
-	}
-	return newRideDeleteEntry(
-		rideUnit{},
-		key,
-	), nil
-}
-
-func checkedIntDataEntry(_ environment, args ...rideType) (rideType, error) {
-	if err := checkArgs(args, 2); err != nil {
-		return nil, errors.Wrap(err, "checkedIntDataEntry")
-	}
-	key, ok := args[0].(rideString)
-	if !ok {
-		return nil, errors.Errorf("checkedIntDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	value, ok := args[1].(rideInt)
-	if !ok {
-		return nil, errors.Errorf("checkedIntDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	return newRideIntegerEntry(
-		key,
-		value,
-	), nil
-}
-
-func checkedStringDataEntry(_ environment, args ...rideType) (rideType, error) {
-	if err := checkArgs(args, 2); err != nil {
-		return nil, errors.Wrap(err, "checkedStringDataEntry")
-	}
-	key, ok := args[0].(rideString)
-	if !ok {
-		return nil, errors.Errorf("checkedStringDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	value, ok := args[1].(rideString)
-	if !ok {
-		return nil, errors.Errorf("checkedStringDataEntry: unexpected argument type '%s'", args[0].instanceOf())
-	}
-	return newRideStringEntry(
-		key,
-		value,
-	), nil
-}
-
 // Constructors
 
 func address(_ environment, args ...rideType) (rideType, error) {
```

Comment:
Why did you leave constructors below (`address`, `alias`, `unit`)? Maybe describe and add them to `pkg/ride/generate/ride_objects.json`?.

Reply:
because these constructors are not relevant and require specific implementations.

Reply:
ok